src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -280,9 +289,10 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 		const editorBorder = 2;
 		const editorPadding = 8;
 		const executeToolbarWidth = 25;
+		const sideToolbarWidth = this.options.renderStyle === 'compact' ? 20 : 0;
 
 		const initialEditorScrollWidth = this._inputEditor.getScrollWidth();
-		this._inputEditor.layout({ width: width - inputPartHorizontalPadding - editorBorder - editorPadding - executeToolbarWidth, height: inputEditorHeight });
+		this._inputEditor.layout({ width: width - inputPartHorizontalPadding - editorBorder - editorPadding - executeToolbarWidth - sideToolbarWidth, height: inputEditorHeight });
```

Comment:
Does the side toolbar just get covered up when not in 'compact' mode? I think it should also be `display: none`

Reply:
or better yet just never create it?